frontend/__tests__/src/components/Metrics/MetricsStep/CycleTime.test.tsx

```diff
@@ -1,8 +1,17 @@
 import { render, within } from '@testing-library/react'
 import { CycleTime } from '@src/components/Metrics/MetricsStep/CycleTime'
 import userEvent from '@testing-library/user-event'
+import { Provider } from 'react-redux'
+import { setupStore } from '../../../utils/setupStoreUtil'
+import { CYCLE_TIME_SETTINGS } from '../../../fixtures'
 
-const title = 'Cycle Time Settings'
+let store = setupStore()
+
+beforeEach(() => {
+  store = setupStore()
+})
+
+const title = CYCLE_TIME_SETTINGS
 const defaultSelected = '----'
```

Comment:
the static variable should be `DEFAULT_SELECTED`
const title = CYCLE_TIME_SETTINGS is not necessary